ruby-service/app/services/database_query_handler.rb

```
@@ -0,0 +1,83 @@
+require 'sqlite3'
```

RuboCop (1.88.2)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)